File: PrintStatTillNo.c
#include<stdio.h>

void Display(int No)
{
    int iCnt = 0;

    iCnt=1;
    while(iCnt <=No)
    {
        printf("Welcome to C programming....\n");
        iCnt++;
    }
}

int main()
{
    int iNo = 0;

    printf("Enter the number of iteration:\n");
    scanf("%d",&iNo);

    Display(iNo);

    return 0;
}

File: PrintStatTillNo.java
import java.util.*;

class PrintStat
{
    public void Display(int No)
    {
        int iCnt = 0;

        iCnt = 1;
        while(iCnt <= No)
        {
            System.out.println("Welcome to java Programming...");
            iCnt++;
        }
    }

}

class PrintStatTillNo
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        int iNo = 0;
        System.out.println("Enter the number of iteration you want");
        iNo = sobj.nextInt();

        PrintStat obj = new PrintStat();

        obj.Display(iNo);
    }
}